### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by mattjmeier**

### CODE REVIEW

File: scripts/write_output_tables.R [hunk 1]
```diff
@@ -39,7 +39,6 @@ for (current_filter in facets) {
   
   prefix <- paste0(params$platform, "_",
                    params$project_title, "_",
-                   paste(params$current_filter, collapse = "_"), "_",
```
Comment (mattjmeier) [inline scripts/write_output_tables.R:42]: This was giving me double underscores... but actually even though they are in subfolders for different facets, they should still retain the $current_filter within the name. Not sure why it isn't getting passed in here via the parameters... Need to troubleshoot that one.

File: scripts/write_output_tables.R [hunk 2]
```diff
@@ -10,7 +10,7 @@ if(is.na(params$group_facet)){
 
 for (current_filter in facets) {
   resultsListAll <- overallResListAll[[current_filter]] 
-  resultsListDEGs <- overallResListAll[[current_filter]] 
+  resultsListDEGs <- overallResListDEGs[[current_filter]] # REVIEW: is this right?
```
Comment (mattjmeier) [inline scripts/write_output_tables.R:13]: This LOOKS like it was a typo, but would definitely like to double check before merging.

Comment (cookkate) [reply scripts/write_output_tables.R:13]: Yeah that looks like it was a copy/paste that didn't get fixed

File: scripts/load-facet-data.R [hunk 3]
```diff
@@ -48,7 +48,9 @@ if(is.na(params$group_facet) && is.na(params$display_group_facet)){
   # case 3: yes facet, yes display facet
 } else if(!is.na(params$group_facet) && !is.na(params$display_group_facet)){
   display_group_filter <- params$display_group_filter
-  
+  # This does not work when the $group_facet and $display_group_facet are different.
+  # It should be possible though... for example, when $display_group_filter includes
+  # more than one facet
```
[the remark this answers is not in the document]
Comment (mattjmeier) [reply scripts/load-facet-data.R:53]: Yeah, I definitely see your point there and agree. Should probably shelve for now.

**Pull request merged**